<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="WIFI">
     <attribute name="title">
      <string>WIFI</string>
     </attribute>
     <widget class="QListWidget" name="listWIFI">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>61</y>
        <width>461</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="WIFIScan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>扫描</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectWIFI">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QLabel" name="WIFIConnect">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>未连接</string>
      </property>
     </widget>
     <widget class="QLabel" name="WIFIName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="netWork1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>设备号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="shebeihao">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manual">
     <attribute name="title">
      <string>手动</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="labVerson">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前版本：
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labVersonNum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
